llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -458,7 +395,7 @@ class MCDCRecordProcessor {
     MCDCRecord::TestVector TV(NumConditions, MCDCRecord::MCDC_DontCare);
 
     // Use the base test vector to build the list of all possible test vectors.
-    buildTestVector(TV);
+    buildTestVector(TV, 1, 0);
```

Comment:
I think default values are acceptable here.

Reply:
If we use default arguments, they will only in this one call site. I think avoiding default arguments is at least equally readable:)

Reply:
I thought default values could make initial values written as inline :)